app/models/colyseus-models/player.ts

```diff
@@ -209,10 +209,12 @@ export default class Player extends Schema implements IPlayer {
 
   updateSynergies() {
     const pokemons: Pokemon[] = values(this.board)
-    const updatedSynergies = computeSynergies(pokemons)
+    var updatedSynergies = computeSynergies(pokemons)
 
     this.updateArtificialItems(updatedSynergies)
 
+    updatedSynergies = computeSynergies(pokemons)
```

Comment:
we should compute twice only if artificial items have been lost, to save some performance. This can be done in another PR if you want